sdks/nodejs/package.json

```diff
@@ -26,5 +26,5 @@
     "publishConfig": {
         "access": "public"
     },
-    "version": "1.20.0"
+    "version": "1.21.0-dev"
```

Comment:
You seem to be missing the matching update to `sdks/nodejs/package-lock.json`. 

Did you update this by hand or run `npm version 1.21.0-dev`?